packages/docs/src/routes/docs/integrations/leaflet-map/index.mdx

```diff
@@ -0,0 +1,68 @@
+---
+title: LeafletJS Map | Integrations
+keywords: 'map, interactive maps'
+contributors:
+  - mugan86
```

Comment:
the PR has this nickname `anartzdev`
is mugan86 correct?

Reply:
mugan86 is my principal gihutb account. If you look at the participants there is mugan86, which is myself but I have two accounts. If it is not correct, I remove mugan86 and put anartzdev, but I am the same person and I prefer to use mugan86 as a contributor